{"name": "test_buurten_failure", "status": "passed", "description": "Failure API request for the buurten", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/gebieden/buurten/0363098000001 to validate whether the no record found with `failure code 404`", "status": "passed", "start": 1646841836302, "stop": 1646841836432}], "start": 1646841836302, "stop": 1646841836432, "uuid": "10e01267-3d43-4c5e-99f7-7bbe232f5c33", "historyId": "6d2d2afe8521e49849a7d1201a789880", "testCaseId": "76f46efd1f9a250d2bb8382d9d249708", "fullName": "e2etest.test_datasetgebiedenbuurten#test_buurten_failure", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetgebiedenbuurten"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "58923-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetgebiedenbuurten"}]}